common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -240,10 +239,9 @@ void set(T value) {
         }
 
         boolean availableToClaim() {
```

Comment:
we cou also rename this method and not have it return a boolean.

Reply:
Yes, and I think the `do...while` loop inside `claim()` is also not needed.